compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skiko.kt

```diff
@@ -312,6 +313,7 @@ open class SkikoComposeUiTest @InternalTestApi constructor(
             platformContext = TestContext(),
             invalidate = { }
         )
+        architectureComponentsOwner.enableSavedStateHandles()
```

Comment:
Could you please add a simple test case that it's available in tests? We already have similar tests in this module